README.md

```diff
@@ -79,7 +79,7 @@ If you prefer to install `deno` in your own base image, you can use the
 ```Dockerfile
 FROM ubuntu
 
-ARG DENO_VERSION=1.14.1
+ARG DENO_VERSION=1.14.2
```

Comment:
Do a full search for 1.14.1. There are a few more places that need updating.

Reply:
There's also a script _update_deno.ts